api/app/wildfire_one.py

```diff
@@ -481,6 +530,50 @@ async def get_hourly_actuals_all_stations(
     return hourly_actuals
 
 
+async def get_wfwx_stations_from_station_codes(session, header, station_codes: Optional[List[int]]):
+    """ Return the WFWX station ids from WFWX API given a list of station codes. """
+    wfwx_stations = await get_stations(session, header, mapper=wfwx_station_list_mapper)
```

Comment:
is this returning only the stations relevant to HFI, or alllll stations in WFWX?

Reply:
It gets them all then filters out the HFI ones we're interested below because WFWX API breaks if you specify too many station parameters.

Reply:
Oh lame. Well I was going to say that requesting weather data for every active wx station in WFWX seems highly excessive, but given that breakage, we'll have to keep the code as is. But I'd strongly recommend inserting a comment here to document that, otherwise it'll be very tempting for a dev to later come along and change the code to try to cut down on the excess.

Reply:
Done